consensus/state_sync/module.go

```diff
@@ -2,15 +2,15 @@ package state_sync
 
 import (
 	"fmt"
-	"log"
 
 	typesCons "github.com/pokt-network/pocket/consensus/types"
+	"github.com/pokt-network/pocket/logger"
 	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
 	"github.com/pokt-network/pocket/shared/modules"
 )
 
 const (
-	DefaultLogPrefix    = "NODE"
+	DefaultLogPrefix    = "NODE-StateSync Module"
```

Comment:
No need for this. When you initiate the logger you pass in the name of the module